.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -10,17 +10,17 @@ body:
 
       For usage questions, please use the following resources:
 
-      - Read the [docs](http://echarts.apache.org/option.html)
+      - Read the [docs](https://echarts.apache.org/option.html)
       - Find in [examples](https://echarts.apache.org/examples/)
       - Look for / ask questions on [Stack Overflow](https://stackoverflow.com/questions/tagged/echarts)
 
-      For non-technical support or general questions, you can email [dev@echarts.apache.org](mailto:dev@echarts.apache.org). And don't forget to subscribe our [mailing list](https://echarts.apache.org/en/maillist.html) to get updated with the project.
+      For non-technical support or general questions, you can email [dev@echarts.apache.org](mailto:dev@echarts.apache.org). And don't forget to subscribe to our [mailing list](https://echarts.apache.org/maillist.html) to get updated with the project.
 
       Also try to search for your issue - it may have already been answered or even fixed in the development branch. However, if you find that an old, closed issue still persists in the latest version, you should open a new issue using the form below instead of commenting on the old issue.
 
 - type: textarea
   attributes:
-    label: What Problem Does This Feature Solve
+    label: What problem does this feature solve?
```

Comment:
It's better to be capitalized title like others here.
I found the `Link to minimal reproduction` and `Any additional comments` section is also not capitalized.

https://capitalizemytitle.com/

Reply:
I personally think a form is just like asking the developer the questions, unlike the title of the paper or another formal one, taking it as a common talk is enough, so no need to capitalize all the first letters.

Reply:
We can use some examples for refrence.

Template in GitHub Doc: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#markdown-issue-template

TypeScript: https://github.com/microsoft/TypeScript/issues/new?assignees=&labels=&template=Bug_report.md&title=

What bothers me is the inconsistency. It looks very random in our template, or in the reference listed above.

Reply:
What I'd like to suggest is as follows,

- Use _**Title case**_ if it's a noun. For example,
    - **Current Behavior**
    - **Expected Behavior**
    - **Steps To Reproduce** 

- Use _**Sentence case**_ if it's a question. For example,
    - **Is there an existing issue for this?**
    - **Anything else?**
    - **What problem does this feature solve?**

Reply:
Looks Good!